src/main.rs

```diff
@@ -105,14 +105,41 @@ fn main() {
             let duration_running = now.duration_since(time_start);
             time = now;
 
+            let duration_last_frame_s = duration_last_frame.as_secs() as f32
```

Comment:
Can leave a FIXME here to change to https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_secs_f32 when it stabilizes in 1.38.0